internal/util/parse.go

```diff
@@ -64,6 +64,21 @@ func ParsePInt64s(ss []string) ([]*int64, error) {
 	return us, nil
 }
 
+// Parses a uint64 from given hex in string.
+func ParseHexUint64s(ss []string) ([]*uint64, error) {
+	us := make([]*uint64, 0, len(ss))
```

Comment:
Could this return a `uint64` slice instead of `*uint64`?

Reply:
It could, I was following the pattern of the other string parsers here.